docs/external-node.md

```diff
@@ -561,3 +562,39 @@ interfaces will be added in the future.
 
 `ExternalNode` name must be unique in the `cluster` scope even though it is
 itself a Namespaced resource.
+
+## Known issues
+
+After installing `antrea-agent` on VM, `antrea-agent` fails to attach the VM's
+network interface to the OVS bridge if the VM is rebooted. On Windows VM,
```

Comment:
I am not sure I understood the issue. After reboot, will agent re-attach interface to OVS, because we do not persist the config to some init file?

Maybe we should say "re-attach" then.

Also feel we may remove "After installing `antrea-agent` on VM", as it does not give useful info.

Reply:
Actually, the issue happens because OVS has persisted the configuration, and OVS is trying to re-configure the ports and interfaces according to OVSDB. But the uplink ($adapterName~) is renamed back as $adapterName by the host after reboot ( this is because we didn't persist the network interface name on the host on Linux), this introduces two issues comparing to the OVSDB configurtion: 1) physical interface (uplink) is not found on the host; 2) internal port is not able to create because of name conflict. 

As for Windows, the reason is different. The uplink and internal port are persistent on the host after reboot, but the openflow entries are not persistent. Hence, the network connectivity is lost. Then antrea-agent fails to start, because it is not able to connect to kube apiserver, so agent can not re-configure the openflow entries to recover the network connectivity.

But both two are because of the original persistent OVSDB configurations. So we give the workaround to remove OVSDB before restart antrea-agent